src/sorter.rs

```diff
@@ -181,9 +198,48 @@ impl<MF, CC: ChunkCreator> SorterBuilder<MF, CC> {
             index_levels: self.index_levels,
             chunk_creator: self.chunk_creator,
             sort_algorithm: self.sort_algorithm,
+            sort_in_parallel: self.sort_in_parallel,
             merge: self.merge,
         }
     }
+
+    /// Creates a [`ParallelSorter`] configured by this builder.
+    ///
+    /// Indicate the number of different [`Sorter`] you want to use to balanced
+    /// the load to sort.
+    pub fn build_in_parallel<U>(self, number: NonZeroUsize) -> ParallelSorter<MF, U, CC>
+    where
+        MF: for<'a> Fn(&[u8], &[Cow<'a, [u8]>]) -> Result<Cow<'a, [u8]>, U>
+            + Clone
+            + Send
+            + 'static,
+        U: Send + 'static,
+        CC: Clone + Send + 'static,
+        CC::Chunk: Send + 'static,
+    {
+        match number.get() {
+            1 => ParallelSorter::Single(self.build()),
+            number => {
```

Comment:
We must do `number - 1` as if we want only 2 threads we will result in the main sending thread and only one sorter which is useless, lets us keep only the single sorter when the number is equal to 2.